### ISSUE TITLE
Pictures and gifs not rendered when exporting to HTML

### ISSUE THREAD

**Issue by abdullaahi140** [typora/typora-issues#1311]
Shows a black box with white cross within it. Alt text is displayed next to it.

**Comment by abnerlee** [comment]
What's your markdown source code contains img looks like?

**Comment by abdullaahi140** [comment]
![screenshot 176](https://user-images.githubusercontent.com/28843151/37398686-33bff714-2777-11e8-95d6-60e54f479b11.png)
![screenshot 175](https://user-images.githubusercontent.com/28843151/37398701-391ea304-2777-11e8-9102-c56527c7b30f.png)

**Comment by abdullaahi140** [comment]
It is possible that the images are not rendered because of `()` in the filename.

However the gif does not have `()` in the filename yet it still does not render.

**Comment by abdullaahi140** [comment]
![screenshot 177](https://user-images.githubusercontent.com/28843151/37398938-fb4b3046-2777-11e8-9b8b-6b1becc14fd0.png)

![screenshot 178](https://user-images.githubusercontent.com/28843151/37398960-0f8721b4-2778-11e8-9837-b397ade92b19.png)

Image rendered. Workaround is to remove `()` from picture filenames.

Gifs still do **not** render.

**Comment by abnerlee** [comment]
Could you update to latest version and confirm this bug again.

I tried on Windows, but image path contains `()` or whitespace works in my testing.

**Comment by abdullaahi140** [comment]
![still](https://user-images.githubusercontent.com/28843151/37471004-b8e2d0bc-2860-11e8-811e-70341f8880a2.png)

![doesn't](https://user-images.githubusercontent.com/28843151/37471028-c41cca28-2860-11e8-8fd0-877d9e88c8fc.png)

![work](https://user-images.githubusercontent.com/28843151/37471048-ce2489f2-2860-11e8-98de-f297097cabb2.png)

Still does not render.

**Comment by abnerlee** [comment]
I see, is it possible to send the exported HTML to us(<hi@typora.io>) ?

**Comment by abdullaahi140** [comment]
Yeah, I'll send it in a few minutes.

**Comment by abdullaahi140** [comment]
@abnerlee I sent the email with all the files I used.

**Comment by abnerlee** [comment]
Can reproduce now. Seems only not work on Edge.

**Comment by abdullaahi140** [comment]
Did you use internet browsers or a PDF application?

**Comment by abdullaahi140** [comment]
Doesn't show in Firefox either. 

![screenshot 189](https://user-images.githubusercontent.com/28843151/37529617-8964a778-292f-11e8-9cf8-12927b6de719.png)

**Comment by abnerlee** [comment]
Should be fixed in new release.

Please reopen if it still exists

**Comment by AlexFreik** [comment]
<img width="1041" alt="Screenshot 2020-12-26 at 21 19 59" src="https://user-images.githubusercontent.com/61039123/103157031-1e258380-47c0-11eb-9649-ef56a7d389ab.png">
<img width="367" alt="Screenshot 2020-12-26 at 21 20 19" src="https://user-images.githubusercontent.com/61039123/103157036-2978af00-47c0-11eb-9afd-ea83dbdd646d.png">

I has this problem at Mac OS Big Sur (updated Typora to the last version)

**Comment by abnerlee** [comment]
Opened in which browser? I can see images from chrome.

**Comment by AlexFreik** [comment]
> Opened in which browser? I can see images from chrome/safari.

I used safari, but it appears to work with chrome, thanks

**Comment by ndricca** [comment]
Hi, I am still encountering this on Firefox:

code:
```markdown
here's a black rectangle:

![image-20220119165518849](C:\Users\jl1andricca\Desktop\image-20220119165518849.png)
```
html not rendering:
```html
<body class='typora-export os-windows'>
    <div class='typora-export-content'>
        <div id='write'  class=''>
            <p><span>here&#39;s a black rectangle:</span></p>
            <p><img src="C:\Users\jl1andricca\Desktop\image-20220119165518849.png" referrerpolicy="no-referrer" alt="image-20220119165518849"></p>
            <p>&nbsp;</p>
        </div>
    </div>
</body>
```
screenshot:
![image](https://user-images.githubusercontent.com/22129594/150169192-53b7fdc2-b1a5-4d27-88c1-9e80e19699a0.png)


I can render the image if I add `file:///` into the absolute path to the image:
```html
<body class='typora-export os-windows'>
    <div class='typora-export-content'>
        <div id='write'  class=''>
            <p><span>here&#39;s a black rectangle:</span></p>
            <p><img src="file:///C:\Users\jl1andricca\Desktop\image-20220119165518849.png" referrerpolicy="no-referrer" alt="image-20220119165518849"></p>
            <p>&nbsp;</p>
        </div>
    </div>
</body>
```
screenshot:
![image](https://user-images.githubusercontent.com/22129594/150169126-2d02563d-b9cd-493e-a6a8-250f6c60498d.png)

My Typora version is 0.11.8 (beta). Thanks!

**Comment by hackthedev** [comment]
From what i encountered its generally best to use relative paths and let typora generate a assets folder as found in the settings. Tho my issue is that when exporting to html i would like it to embed the image from the assets folder into base64 or something so it will always work.

Sadly it seems like the Export option for HTML doesnt have such feature
